libmscore/notifier.hpp

```diff
@@ -114,6 +114,8 @@ Listener<Data>& Listener<Data>::operator=(Listener<Data>&& other)
             }
       else
             setNotifier(nullptr);
+
+      return *this;
```

Comment:
we really do have an .hpp file? Seems very unusual and might warrant to get changed to .h

Reply:
may not be the score for this PR though

Reply:
There, the entire class is implemented in an .h file. I personally do not like the implementation in the .h file, but I think the correction of this case needs to be done separately.

Reply:
This is a template class, we have no real choice here. Moving it to `.cpp` will require explicit declaration of this template specializations in that `.cpp` file which makes the code somewhat less modular.

Reply:
Right !!

Reply:
I don't even see that file in QtCreator in the Project tree, neither under libmscore > Header Files nor under libmscore > Source files

Reply:
Same in Visual Studio

Reply:
It is the only hpp file in the entire source tree, except for a few in thirdparty/dtl/